### PR TITLE
Vpsadev 206

### PR DESCRIPTION
**Pull request by cynmij**

### CODE REVIEW

File: modules/stanford_news_views/stanford_news_views.views_default.inc [hunk 1]
```diff
@@ -529,7 +529,6 @@ function stanford_news_views_views_default_views() {
   $handler->display->display_options['fields']['field_s_image_info']['field'] = 'field_s_image_info';
   $handler->display->display_options['fields']['field_s_image_info']['label'] = '';
   $handler->display->display_options['fields']['field_s_image_info']['exclude'] = TRUE;
-  $handler->display->display_options['fields']['field_s_image_info']['alter']['make_link'] = TRUE;
```
Comment (sherakama) [inline modules/stanford_news_views/stanford_news_views.views_default.inc]: Did you intend to remove the link on this field?

File: modules/stanford_news_views/stanford_news_views.views_default.inc [hunk 2]
```diff
@@ -776,7 +786,6 @@ function stanford_news_views_views_default_views() {
...
```
Comment (sherakama) [inline modules/stanford_news_views/stanford_news_views.views_default.inc]: and this one.

File: modules/stanford_news_views/stanford_news_views.views_default.inc [hunk 3]
```diff
@@ -990,7 +1010,6 @@ function stanford_news_views_views_default_views() {
...
```
Comment (sherakama) [inline modules/stanford_news_views/stanford_news_views.views_default.inc]: and this one.

**Comment by cynmij** [comment]
Yes.  I plan to wrap the image and title fields with the more info link in another ticket.

**Comment by sherakama** [comment]
Ok, this is great. Thanks.

**Pull request merged**